src/index.js

```diff
@@ -520,13 +520,13 @@ module.exports = {
 			}
 
 			// Rate limiter
-			if (route.rateLimit) {
-				const opts = route.rateLimit;
-				const store = route.rateLimit.store;
+			if (route.opts.rateLimit) {
+				const opts = route.opts.rateLimit;
+				const store = route.opts.rateLimit.store;
 
 				const key = opts.key(req);
 				if (key) {
-					const remaining = opts.limit - await store.inc(key);
+					const remaining = opts.limit - store.inc(key);
```

Comment:
why did you remove await ? It will remove the async store ( and so, the feature I add in https://github.com/moleculerjs/moleculer-web/pull/317 )

Reply:
sorry , i didn't see the await in the master branch. I will add it , apologies.